### DISCUSSION TITLE
Add "bug report" to discussion categories

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#13] [New Feature Requests]
Right now, there's only "feature request" and "q&a" on this feedback repo.

Sometimes users will want to submit a bug report, so a new category is appropriate.

**Comment by hans-cyfrin** [comment]
@aviggiano 
We added a new category [Bug Report](https://github.com/orgs/solodit/discussions/categories/bug-report).
Thank you for your suggestion.

**Comment by aviggiano** [comment]
Thanks!!
